code/modules/mob/living/simple_animal/hostile/carp.dm

```diff
@@ -59,15 +60,34 @@
 	icon_living = "megacarp"
 	icon_dead = "megacarp_dead"
 	icon_gib = "megacarp_gib"
-	maxHealth = 65
-	health = 65
+	maxHealth = 20
+	health = 20
 	pixel_x = -16
 	mob_size = MOB_SIZE_LARGE
 
 	obj_damage = 80
 	melee_damage_lower = 20
 	melee_damage_upper = 20
 
+	var/regen_cooldown = 0
+
+/mob/living/simple_animal/hostile/carp/megacarp/Initialize()
+	. = ..()
+	name = "[pick(GLOB.megacarp_first_names)] [pick(GLOB.megacarp_last_names)]"

```

Comment:
You know instead of having every single last name end in shark you could have just added it here so it's ` "[pick(GLOB.megacarp_first_names)] [pick(GLOB.megacarp_last_names)] Shark"` lol

Reply:
Sharknado, Shawshark, Mega and minilodon.

Reply:
how could i have missed!